data/data/bootstrap/files/usr/local/bin/installer-masters-gather.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Should we also be collecting any manifests provided by the installer? 
I know the install dir (locally) should have this but do we want to get it from the masters to possibly compare against?

Reply:
> I know the install dir (locally) should have this but do we want to get it from the masters to possibly compare against?

Not in this PR please, lets just get this out without creeping scope.

Reply:
All the bootstrap manifests are in `.openshift_install_state.json` in bootstrap section, I don't think it worth duplicating these

Reply:
I was not aware of that. It's good to know when we document this, for support.